### PR TITLE
Feat/implement company page

### PR DESCRIPTION
**Pull request by pikann**

### CODE REVIEW

File: src/pages/common/Company.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                    </li>
+                )
+            })
+        }
+        return result;
+    }
+}
+
+
+export default Company;
```
Comment (hieud1245k) [inline src/pages/common/Company.js:124]: missing an empty line in the end of file

File: src/style.css [hunk 2]
```diff
@@ -264,4 +264,25 @@ span.icon2 {
...
+
+.company-logo img{
+    top: 50%;
+    -ms-transform: translateY(-50%);
+    transform: translateY(-50%);
+    position: absolute;
+}
+
+.company-card { transition: all .5s ease-in-out; }
+.company-card:hover { transform: scale(1.04); }
```
Comment (hieud1245k) [inline src/style.css:288]: .

File: src/pages/common/Company.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        this.setPage(1);
+    }
+    setPage = (page) => {
+        this.setState({"page": page});
+        this.setState({"list_employer": []});
+        this.setListCompany(page);
+    }
+    setListCompany = (page) => {
+        this.setState({loading: true});
+        callApi("employer?name=" + this.state.name + "&page=" + (page - 1), 'GET').then(rs => {
```
Comment (hieud1245k) [inline src/pages/common/Company.js:29]: define a format string

File: src/pages/common/Company.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+            this.setState(rs.data);
+            this.setState({loading: false});
+        });
+    }
+    setCountPage = () => {
+        callApi("/employer/page?name=" + this.state.name, 'GET').then(rs => {
```
Comment (hieud1245k) [inline src/pages/common/Company.js:35]: .

File: src/pages/common/Company.js (hunk 3 above)
Comment (hieud1245k) [inline src/pages/common/Company.js:29]: define a format string

File: src/pages/common/Company.js (hunk 4 above)
Comment (hieud1245k) [inline src/pages/common/Company.js:35]: .

File: src/style.css (hunk 2 above)
Comment (hieud1245k) [inline src/style.css:288]: .

File: src/pages/common/Company.js (hunk 1 above)
Comment (hieud1245k) [inline src/pages/common/Company.js]: missing an empty line in the end of file

File: src/style.css (hunk 2 above)
Comment (hieud1245k) [inline src/style.css]: .

File: src/pages/common/Company.js (hunk 3 above)
Comment (pikann) [reply src/pages/common/Company.js:29]: In Reactjs, format string need to import library. I think it's not effective in this case.

**Pull request merged**